### DISCUSSION TITLE
Disable Live-Mode and Free Form Text Search

### DISCUSSION THREAD

**Discussion by rja1** [provectus/kafka-ui#2959] [Q&A] [answered]
Just wondering if there's a way to disable these features.

The issue is that we have topics with 1+ billion messages.  Using Live Mode on a high volume topic, will just tank your browser (or the container).  The free form text search is problematic as it'll literally scan all billion events, looking for the text string.  This spikes I/O, load averages and impacts the performance and health of the brokers.

Please advise.

Thanks,
Rob

**Comment by Haarolean** [comment, marked as answer]
Hey, currently there's no feature like this. We'll take this into consideration in future releases.

**Reply by rja1** [reply]
Sounds good, thanks @Haarolean

**Reply by iliax** [reply]
@rja1 we now reviewing PR https://github.com/provectus/kafka-ui/pull/2981 that can fix your problem. You will be able to set max per-cluster polling rate (to avoid IO spikes while reading the topic) and also in tailing mode UI production rate is limited to 20msg/sec

**Reply by rja1** [reply]
Excellent, thanks @iliax!!!
